modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployResourceEntity.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.database.service;
+
+import jakarta.persistence.CollectionTable;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MapKeyColumn;
+import jakarta.persistence.Table;
+import java.util.Map;
+import lombok.Data;
+import org.eclipse.xpanse.modules.database.common.DeployResourceKind;
+
+/**
+ * DeployResourceEntity for persistence.
+ */
+@Data
+@Table(name = "deployResource")
```

Comment:
just to stick to naming conventions, please make the table name full uppercase. 
DEPLOY_RESOURCE

Reply:
okie, I have some database trouble here, let's fix this in the next PR.